pkg/sdn/plugin/ovscontroller.go

```diff
@@ -306,8 +306,16 @@ func (oc *ovsController) SetPodBandwidth(hostVeth string, ingressBPS, egressBPS
 	return nil
 }
 
-func getPodDetailsByContainerID(flows []string, containerID string) (int, string, string, string, error) {
-	note, err := getPodNote(containerID)
+func getPodDetailsBySandboxID(ovsif ovs.Interface, sandboxID string) (int, string, string, string, error) {
+	flows, err := ovsif.DumpFlows()
```

Comment:
Currently DumpFlows is used in during startup, update pod and teardown case where podIP is empty. We can't optimize during startup, update pod may not be a frequent operation. May be we can introduce this idea when dumpFlows() is called frequently.